frontend/src/app/(navfooter)/settings/editor/EditorSettings.module.scss

```diff
@@ -0,0 +1,5 @@
+.loading {
+    display: flex;
+    gap: 0.5em;
+    padding: 1em;
+}
```

Comment:
Probably easier to use Tailwind `flex gap-2 p-4`

Reply:
fair 'nough